examples/ondisk.py

```diff
@@ -118,7 +118,7 @@
 
     >>> with var.get_local_path() as f:
     ...     with xr.open_dataset(f) as ds:
-    ...         print(ds) # doctest: +ELLIPSIS
+    ...         print(ds) # doctest: +SKIP
```

Comment:
This test would not pass with ``doctest: +ELLIPSIS`` 
https://travis-ci.org/ClimateImpactLab/DataFS/jobs/219518927